cognite/client/_api/labels.py

```diff
@@ -156,3 +156,31 @@ def delete(self, external_id: str | SequenceNotStr[str] | None = None) -> None:
                 >>> client.labels.delete(external_id=["big_pump", "small_pump"])
         """
         self._delete_multiple(identifiers=IdentifierSequence.load(external_ids=external_id), wrap_ids=True)
+
+    def retrieve_multiple(
```

Comment:
Should we call this just `retrieve` and use overloads to type annotate single (returns `LabelDefinition`) vs multiple (returns `LabelDefinitionList`)?

Reply:
hmm could be, do you know an example in the SDK somewhere that does the same?

Reply:
Here's the list (essentially all newly-ish added ones):
```
- client.data_modeling.data_models.retrieve
- client.data_modeling.containers.retrieve
- client.data_modeling.spaces.retrieve
- client.iam.sessions.retrieve
- client.sequences.data.retrieve
- client.units.retrieve
- client.iam.user_profiles.retrieve
- ```

Reply:
@haakonvt fixed